docs/maya/ui_overview.md

```diff
@@ -162,12 +164,21 @@ Option boxes can be used to launch a UI to assign the name to the deformer and c
 - **Connect Activations Plug**. Connects the output compression map from an AdnEdgeEvaluator node to the activations plug of the AdnSimshape selected deformer. The selection must consist of: first the Edge Evaluator node and lastly the AdnSimshape deformer node.
 - **Disconnect Activations Plug**. Disconnects the output compression map from an AdnEdgeEvaluators node from the activations plug of the AdnSimshape selected deformer. The selection must consist of: first the Edge Evaluator node and lastly the AdmSimshape deformer node.
 
+#### Skin
+
+<figure style="width: 50%;" markdown>
```

Comment:
Are we showing examples of multiple targets being added to the skin simulation? For example showing biped mike with N targets and then the fascia. I've not compiled the docs yet so I might see it once I serve the mkdocs

Reply:
No, we are not adding any new screenshots/examples of a new setup using multiple targets. Maybe we can add a subsection or something in "A Simple Setup" page at the end of the AdnSkin section.

Reply:
I'm leaving this open so @carlosmonteagudoinbibo can decide. Thanks!

Reply:
I would leave "A Simple Setup" as it is. Maybe, we can put an example of multi targets in the Advanced Section. A screenshot that shows:
- outliner having muscles separated and selected
- the viewport with the muscles and the fascia selected
- and one of this options: the cursor (almost hovering) the add targets shelf button (almost hovering because of the background problem); or the cursor hovering the "Add Targets" option from the AdonisFX menu

Reply:
Agreed!

Reply:
Added that screenshot at the end of the skin page.

Reply:
Brilliant! Thanks!